src/main/java/org/edumips64/ui/swing/SplashScreen.java

```diff
@@ -0,0 +1,49 @@
+package org.edumips64.ui.swing;
```

Comment:
Can you please add the standard header (copyright, license, author) to this file?

Reply:
Done!